feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/section/TimetableList.kt

```diff
@@ -128,13 +134,21 @@ fun TimetableList(
                 Column {
                     timetableItems.forEach { timetableItem ->
                         val isBookmarked = uiState.timetable.bookmarks.contains(timetableItem.id)
+                        val haptic = LocalHapticFeedback.current
                         TimetableListItem(
                             timetableItem,
                             modifier = Modifier
                                 .clickable { onTimetableItemClick(timetableItem) }
                                 .padding(top = 10.dp),
                             isBookmarked = isBookmarked,
-                            onBookmarkClick = onBookmarkClick,
+                            onBookmarkClick = { timetableItem, isBookmarked ->
+                                haptic.performHapticFeedback(
+                                    HapticFeedbackType(
+                                        HapticFeedbackConstants.LONG_PRESS,
```

Comment:
May I ask why you used LONG_PRESS here? 🙋

Reply:
@takahirom 
it was supported starting from API Level 3, and the user experience and strength of haptic feedback were general! and It looks like it's long because it written Long_PRESS, but it's often used for general touch as well.

Do you have any better ideas ?!

Reply:
I'm just not familiar with this API and I was just wondering because this click is not a long press. Is there any API we can use for this? 👀

Reply:
How about [VIRTUAL_KEY](https://developer.android.com/reference/android/view/HapticFeedbackConstants#VIRTUAL_KEY
) or [KEYBOARD_TAP](https://developer.android.com/reference/android/view/HapticFeedbackConstants#KEYBOARD_TAP) ?!

Reply:
@kosenda @fumiya-kume 
May I ask your opinion on this?

Reply:
Personally, I think `LONG_PRESS` is fine. 
Also, this may be off topic, but I thought Haptics was unnecessary when removing bookmarks.

Reply:
Oh, that's a good idea.
4330d478f1f7715c612d7d2f0c8b520c9ee4e54e